javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -62,13 +62,15 @@ public Javers javers() {
     }
 
     @Bean(name = "authorProvider")
+    @ConditionalOnMissingBean
```

Comment:
uproszczę to jednak tylko do:
```
@Bean(name = "authorProvider")
@ConditionalOnMissingBean
public AuthorProvider springSecurityAuthorProvider() {
         return new SpringSecurityAuthorProvider();
}
```
i wywalę tą mockową implementację, nie widzę zagrożeń, implementacja SpringSecurityAuthorProvider jest odporna na 'brak' security contextu